android/src/main/kotlin/io/dolby/comms/sdk/flutter/events/RecordingEventEmitter.kt

```diff
@@ -0,0 +1,20 @@
+package io.dolby.comms.sdk.flutter.events
+
+import com.voxeet.sdk.json.RecordingStatusUpdatedEvent
+import com.voxeet.sdk.models.v1.RecordingStatus
+import org.greenrobot.eventbus.Subscribe
+import org.greenrobot.eventbus.ThreadMode
+
+class RecordingEventEmitter(eventChannelHandler: EventChannelHandler) : NativeEventEmitter(eventChannelHandler) {
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    fun onRecordingStatusUpdate(event: RecordingStatusUpdatedEvent) {
+        mapOf(
+            "recordingStatus" to "EVENT_${event.recordingStatus}",
```

Comment:
It should be `"recordingStatus" to event.recordingStatus`

Reply:
changed